projects/v2/src/app/services/sheet.service.ts

```diff
@@ -55,4 +55,14 @@ export class SheetService {
   updatePlaygroundData(data: string[][]) {
     return this.http.post(`${URL}/v2/playground`, {data});
   }
+
+  /**
+   * Service to fetch the data for a sheet from CSV file using the api
+   * @param url is the link to the csv file of the sheet
+   */
+   fetchDataFromCSV(url: string) {
+    return this.http.post(`${URL}/v2/getDataFromCSV`, { data: {
```

Comment:
sheetUrl doesn't need to be inside of a data object. Just `{ csvUrl: url }` for the second argument.

Reply:
Ok Sure